app/uk/gov/hmrc/cgtpropertydisposalsfrontend/models/audit/SubscriptionRequestEvent.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cgtpropertydisposalsfrontend.models.audit
+
+import play.api.libs.json.{Json, OFormat}
+import uk.gov.hmrc.cgtpropertydisposalsfrontend.models.address.Address
+
+final case class SubscriptionRequestEvent(
+  prePopulatedUserData: Option[PrePopulatedUserData],
+  manuallyEnteredData: Option[ManuallyEnteredData]
+)
+
+final case class PrePopulatedUserData(
+  regime: String,
+  sapNumber: String,
+  individualDetails: Option[IndividualAuditDetails],
+  trustDetails: Option[TrustAuditDetails],
+  emailAddress: EmailAuditDetails
```

Comment:
again, this should be optional and the same thing for the manually entered data

Reply:
From what I understand from your confluence page, your comments here, and your comments on slack, is that either we audit prepop data or manually entered data but not both. If in the manual entry payload only some of the data has been manually entered, what do you want to happen in that scenario? Say only the contact name was manually entered but the email wasn't.